src/openapi/common-responses.ts

```diff
@@ -0,0 +1,74 @@
+import { OpenAPIV3 } from 'openapi-types';
+
+export const NOT_READY_MSG =
+    'The Unleash Proxy has not connected to the Unleash API and is not ready to accept requests yet.';
+
+export const notReadyResponse: OpenAPIV3.ResponseObject = {
+    description: "The proxy isn't ready to accept requests yet.",
+    content: {
+        'text/plain': {
+            schema: {
+                type: 'string',
+                example: NOT_READY_MSG,
+            },
+        },
+    },
+} as const;
```

Comment:
Do we need the `as const`s?

Reply:
Probably not. But does it do any harm? It does guarantee that you can't change the content, so I'm leaning towards allowing it.